packages/api/src/routes/internal.ts

```diff
@@ -100,7 +104,7 @@ router.post(
       });
       result[org.cxId] = orgRes;
     }
-    return res.json(result);
+    return res.json({ result });
```

Comment:
Just curious, any particular reason to update this?

Reply:
Oh.. this is a left-over from when I was sending more things in the return and decided to retract those changes..